documentation/Demo-Server-Guide.md

```diff
@@ -160,6 +160,7 @@ sudo chown 472 grafana_on_port_8009
 export TAG='front-demo-19-08-2021_back-demo-19-08-2021_pg-12_node-14'
 export SMTP_SECRET='add_smtp_secret_here'
 export WEB_URL='https://irims-demo.msupply.org:<replace port>'
+export JWT_SECRET='random private key'
```

Comment:
Is this where we set the new JWT for this server? Maybe worth mentioning how to generate this JWT and use here. Do we need to keep this saved somewhere for future use? Like a Bitwarden entry?

Reply:
I don't think it really matters, we can just make up a new one every time we launch a new instance (or even restart the server). The only time we need to keep this value is if we want existing JWTs to remain valid (like in our testing environment, or with existing Postman collections, say)

But yes, have added a note 084b0c65